### PR TITLE
add a how to to the landing page

### PR DESCRIPTION
**Pull request by johngruen**
#606

### CODE REVIEW

File: testpilot/frontend/static-src/app/templates/landing-page.js [hunk 1]
```diff
@@ -64,24 +65,39 @@ export default `
...
-              <div data-hook='experiment-list'></div>
-          </div>
-          <div class="blue">
-            <footer id="main-footer" class="responsive-content-wrapper">
-              <div class="centered-banner">
-                <div class="copter-wrapper">
-                  <div class="copter"></div>
+            <div class="transparent-container">
+              <div class="responsive-content-wrapper delayed-fade-in">
+                  <h2 class="card-list-header">Try these features today with Test Pilot</h2>
```
Comment (pdehaan) [inline testpilot/frontend/static-src/app/templates/landing-page.js]: Missing l10n id?

File: testpilot/frontend/static-src/images/Untitled-2.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...,0,0-.88,1h0a14.61,14.61,0,0,0-3.61-.47,11.91,11.91,0,0,0-3.75.59,12,12,0,0,1-2.5-2.88l0,0s-0.06.19-.1,0.19l0,0A9.33,9.33,0,0,1,7.4,6.5c0-.11,0-0.21,0-0.31a2.67,2.67,0,0,0-.78.92,6,6,0,0,0-.53,1.23C6,8.64,5.9,8.82,5.84,9h0s0-.5,0-0.5h0a4.15,4.15,0,0,0-.48.92,5.27,5.27,0,0,0-.23.82v0a2.31,2.31,0,0,0,0-.36h0a15.6,15.6,0,0,0-.63,2.19,13.09,13.09,0,0,0-.34,2.08,20.44,20.44,0,0,0,0,3.09c0,0.18,0,.35,0,0.51a14.07,14.07,0,0,0-1.44,2.51A34.16,34.16,0,0,0,0,29.58,22.31,22.31,0,0,1,1.54,26a0.38,0.38,0,0,1,0,.07,33,33,0,0,0-1,11.76,31.56,31.56,0,0,1,.59-3.55,30.18,30.18,0,0,0,4.1,13.38,27.08,27.08,0,0,0,7.32,8.51c0.51,0.42,1,.82,1.56,1.21,3.41,2.56,8.72,5.7,13.24,6.38a6.9,6.9,0,0,1-2.56-1.41A34.23,34.23,0,0,0,33.6,64h0.5a1.79,1.79,0,0,1-1.48-.82s0.52,0,1.4,0c3.18,0,11-.4,15.58-4.09A6.33,6.33,0,0,0,52,55.84a23.26,23.26,0,0,0,5.46-4.13,12.15,12.15,0,0,0,3.84-7.9c0,0.07,0,.14,0,0.21a3.5,3.5,0,0,0-.3-2.3C62.73,38.12,65.4,30,63.12,23.38l0,0.2a1.11,1.11,0,0,0,0-.12s-0.43,1.33-1,3.38c0-.27-0.08-0.55-0.13-0.82a30.46,30.46,0,0,0-.39-7.72,15.57,15.57,0,0,0-5.41-8.66l0.25,0.22-0.3-.23s0.07,0.37.18,1a12.14,12.14,0,0,0-4.38-4.13S52,7,52.14,8A30.11,30.11,0,0,0,31.75,0h0Z" transform="translate(0 0)"/></svg>
```
Comment (pdehaan) [inline testpilot/frontend/static-src/images/Untitled-2.svg]: Needs better file name than untitled?

Comment (pdehaan) [inline testpilot/frontend/static-src/images/Untitled-2.svg]: Not sure if this is used, actually.

Comment (johngruen) [inline testpilot/frontend/static-src/images/Untitled-2.svg]: ah, yeah that slipped through the cracks

**Pull request merged**